docs/internal_api/index.rst

```diff
@@ -188,6 +188,15 @@ Accurate Computing Utils
    utils.computing._fast_two_sum
    utils.computing._two_sum
    utils.computing._two_prod_fma
+   utils.computing._comp_prod_FMA
+   utils.computing._sum_of_squares_re
+   utils.computing._vec_sum
+   utils.computing._norm_faithful
+   utils.computing._normL
+   utils.computing._normG
```

Comment:
Can you rename this without the capital letters at the end (to keep them pythonic)

Reply:
The capital letters are the way to distinguish the different algorithm(With different performance and accuracy). And the names are from the papers (They are using MATLAB). I am trying to adhere the naming from the papers because 1. These are commonly recognized as the name for those operations. 2. It's easy for us to refer the algorithm back to the paper. But I can rename it as `_norm_l` and `_norm_g`, does that sounds ok with you?

Reply:
The lowcase with an underscore is perfect